src/windy/platforms/win32/windefs.nim

```diff
@@ -21,6 +21,7 @@ type
   DWORD* = int32
   LPCSTR* = cstring
   LPSTR* = cstring
+  SIZE_T* = int64 # only for 64 bit
```

Comment:
Is size_t a windows thing or a C thing? If its a C thing maybe just use int64 directly?